### DISCUSSION TITLE
Discussion: This is awesome

### DISCUSSION THREAD

**Discussion by kokroo** [Navideck/universal_ble#7] [General]
Just wanted to say this is awesome, we finally have something that covers BLE across all Flutter platforms!

Is there any roadmap or stability matrix yet?

**Comment by fotiDim** [comment]
@kokroo glad you liked it! All platforms except Windows and Linux are fully functional.
Windows is generally stable. We have only spotted some performance improvements that could be done when discovering devices, nothing that is a blocker for using it in production.
Linux is currently being tested. It will generally work but there might be bugs.

**Reply by kokroo** [reply]
@fotiDim If this works well across all platforms, we will be using it in production for about 1 million users. I will also try to get some donations your way to help you with the great work!

**Reply by fotiDim** [reply]
@kokroo we have released 0.9.1. Linux should be also stable now. We have successfully tested it in one of our apps using amd64 and arm64 Ubuntu. Feel free to take it for a spin and see if it suits your use case.

If someone wants to donate for our work, he can do it through [GitHub Sponsors](https://github.com/sponsors/Navideck). We would be most grateful 🙏.

**Reply by kokroo** [reply]
@fotiDim Thanks for the heads-up, it will be at least 1 month if not more before we start integration tests with this library. I am excited to take it for a spin and report back!

**Reply by dashpodfirm** [reply]
There is some issue in universal_ble it is not giving complete advertisement packet,is there any way to get complete advertisement packet

**Comment by dashpodfirm** [comment]
I was working on universal_ble plugin and i am not able to get complete advertisement data,is there any way to get it?

**Reply by rohitsangwan01** [reply]
@dashpodfirm you can open a new issue, with details like, 
What are you expecting and what are you getting
Platform and Platform Version
please also attach logs
